[test-triage] mask_rom_e2e_bootup_success

### Hierarchy of regression failure

Chip Level

### Failure Description

Test mask_rom_e2e_bootup_success has 1 failures.

0.mask_rom_e2e_bootup_success.3436865675
Log /container/opentitan-public/scratch/os_regression/chip_earlgrey_asic-sim-vcs/0.mask_rom_e2e_bootup_success/out/run.log

  Loading: 0 packages loaded
  Loading: 0 packages loaded
  ERROR: Skipping '//sw/device/silicon_creator/mask_rom:e2e_bootup_success_sim_dv': no such target '//sw/device/silicon_creator/mask_rom:e2e_bootup_success_sim_dv': target 'e2e_bootup_success_sim_dv' not declared in package 'sw/device/silicon_creator/mask_rom' defined by /workspace/mnt/repo_top/sw/device/silicon_creator/mask_rom/BUILD
  WARNING: Target pattern parsing failed.
  ERROR: no such target '//sw/device/silicon_creator/mask_rom:e2e_bootup_success_sim_dv': target 'e2e_bootup_success_sim_dv' not declared in package 'sw/device/silicon_creator/mask_rom' defined by /workspace/mnt/repo_top/sw/device/silicon_creator/mask_rom/BUILD
  INFO: Elapsed time: 23.264s
  INFO: 0 processes.
  FAILED: Build did NOT complete successfully (1 packages loaded)
  FAILED: Build did NOT complete successfully (1 packages loaded)
  make: *** [/workspace/mnt/repo_top/hw/dv/tools/dvsim/sim.mk:73: sw_build] Error 1

### Steps to Reproduce

- GH revision: [9c0b24ddb](https://github.com/lowrisc/opentitan/tree/9c0b24ddbeb5616bd8db661ec1395f2d4f326bc4)
- util/dvsim/dvsim.py hw/top_earlgrey/dv/chip_sim_cfg.hjson -i mask_rom_e2e_bootup_success --build-seed 2483310614 --fixed-seed 3436865675

### Tests with similar or related failures

_No response_

--- a/hw/top_earlgrey/dv/chip_rom_tests.hjson
+++ b/hw/top_earlgrey/dv/chip_rom_tests.hjson
@@ -11,7 +11,7 @@
     {
       name: rom_e2e_bootup_success
       uvm_test_seq: chip_sw_base_vseq
-      sw_images: ["//sw/device/silicon_creator/rom:e2e_bootup_success:1:signed"]
+      sw_images: ["//sw/device/silicon_creator/rom/e2e:e2e_bootup_success:1:signed"]
       en_run_modes: ["sw_test_mode_rom"]
       run_opts: ["+sw_test_timeout_ns=20000000"]
     }

--- a/sw/device/silicon_creator/rom/e2e/BUILD
+++ b/sw/device/silicon_creator/rom/e2e/BUILD
@@ -5,6 +5,7 @@
 load(
     "//rules:opentitan_test.bzl",
     "cw310_params",
+    "dv_params",
     "opentitan_functest",
     "verilator_params",
 )
@@ -22,6 +23,9 @@ opentitan_functest(
         bitstream = "//hw/bitstream:rom",
         exit_failure = BOOT_FAILURE_MSG,
         exit_success = BOOT_SUCCESS_MSG,
+    ),
+    dv = dv_params(
+        rom = "//sw/device/silicon_creator/rom",
     ),
     manifest = "//sw/device/silicon_creator/rom_ext:manifest_standard",
     signed = True,

